public/app/features/dashboard-scene/panel-edit/VizPanelManager.tsx

```diff
@@ -217,6 +218,14 @@ export class VizPanelManager extends SceneObjectBase<VizPanelManagerState> {
     runner.setState({ queries });
   }
 
+  public changeTransformations(transformations: DataTransformerConfig[]) {
+    const dataProvider = this.state.panel.state.$data;
+    if (dataProvider instanceof SceneDataTransformer) {
+      dataProvider.setState({ transformations });
+      dataProvider.reprocessTransformations();
+    }
```

Comment:
Hmm, think that would be nice to be handled in the SceneDataTransformer state change handler? WDYT @torkelo ?

Reply:
yea, maybe but not sure it's needed. It's not that common yet to update transformations, and having this manual call feels ok, it's similar with SceneQueryRunner in that you have to call runQueries after updating queries